deployment/migration-assistant-solution/lib/solutions-stack.ts

```diff
@@ -132,8 +161,37 @@ export class SolutionsInfrastructureStack extends Stack {
             role: bootstrapRole
         })
 
+        let vpc;
+        if (props.createVPC) {
+            vpc = new Vpc(this, 'Vpc', {});
+        }
+        else {
+            const vpcIdParameter = new CfnParameter(this, 'VPCId', {
+                type: 'AWS::EC2::VPC::Id',
+                description: 'Specify the VPC id to place Migration resources into'
+            });
+            addParameterLabel(parameterLabels, vpcIdParameter, "VPC Id")
```

Comment:
Lets drop the word 'id'
```suggestion
            addParameterLabel(parameterLabels, vpcIdParameter, "VPC")
```